source/monitoring.txt

```diff
@@ -12,8 +12,142 @@ Monitoring and Logging
+as {+jconsole+}.
+
+The {+mkc+} provides metrics for individual connector **tasks**.
+Tasks are classes instantiated by a connector that copy data to and
+from data stores and {+ak+}. A connector manages one or more tasks.
+
+To learn more about JMX, see the following resources from Oracle:
+
+- `Java Management Extensions Guide <https://docs.oracle.com/en/java/javase/17/jmx/introduction-jmx-technology.html>`__
+- `Using {+jconsole+} <https://docs.oracle.com/en/java/javase/17/management/using-jconsole.html>`__
+
+To learn more about tasks in {+kc+}, see the following resources:
+
+- `Kafka Connect Concepts from Confluent <https://docs.confluent.io/platform/current/connect/concepts.html#tasks>`__
+- `{+kc+} API documentation for Task interface <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/connector/Task.html>`__
+
+Enable Monitoring
+~~~~~~~~~~~~~~~~~
+
+The {+mkc+} uses {+kc+}'s metrics infrastructure to
+serve metrics. To read the metrics produced by your connector,
+enable JMX in your {+kc+} deployment.
+
+To learn how to enable JMX for a {+kc+} instance
+running on your host machine, see the
+`Official Kafka Documentation <https://kafka.apache.org/documentation/#monitoring>`__.
+
+To learn how to enable JMX for a containerized {+kc+}
+deployment, see
+`Kafka Monitoring and Metrics Using JMX with Docker <https://docs.confluent.io/platform/current/installation/docker/operations/monitoring.html>`__.
+
+.. _kafka-monitoring-example:
+
+Example - Monitor the Quick Start
+---------------------------------
+
+The sample environment provided in the Quick Start exposes metrics
+on your host machine at the URI ``localhost:35000``.
+
+To view these metrics with {+jconsole+}, perform the following actions:
+
+.. procedure::
+   :style: normal
+
+   .. step:: Download {+jconsole+}
+
+      {+jconsole+} is part of the {+java-se+}. To download {+jconsole+}, download the
+      `Java SE Development Kit <https://www.oracle.com/java/technologies/downloads/>`__
+      from Oracle.
+
+   .. step:: Start The Quick Start Pipeline and Add Connectors
+
+      Follow the Quick Start until the :ref:`<kafka-quick-start-send-a-document>`
+      step.
+
+   .. step:: Start {+jconsole+}
+
+      Run the following command from your command line to start {+jconsole+}:
+
+      .. code-block:: shell
+
+         {+jconsole_command+}
+
+   .. step:: Connect to {+kc+} JMX Server
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Enter Your JMX Server URI
+
+            Enter the URI ``localhost:35000`` into the :guilabel:`Remote Process`
+            text input box in the {+jconsole+} interface.
+
+         .. step:: Click :guilabel:`Connect`
+
+         .. step:: Click :guilabel:`Insecure Connection` in the dialog box
+
+   .. step:: Explore Metrics
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Navigate to the :guilabel:`MBeans` tab in {+jconsole+}
+
+         .. step:: Inspect connector metrics.
```

Comment:
Issue: This is the only `step` that ends in a period. Per the [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/tasks/#use-imperative-sentences), that seems to be correct--but it means the other steps should change.

Suggestion: All steps on the page should be written as "complete and correctly punctuated imperative sentence[s]". I think that also means that you shouldn't use title capitalization (e.g., "add connectors" instead of "Add Connectors"). Please let me know if I'm misreading something here.

Reply:
Removed title casing. Added periods. Tried to make steps that weren't complete sentences into complete sentences.